src/components/SlideHover/SlideHover.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './SlideHover.module.scss';
+
+const ProjectScene = ({ children, className }) => {
```

Comment:
we shouldn't pass classname as a prop. All styles configurations should be passed as the props, and modify styles inside the component, e.g.

Button.js
```
function Button({children, isBlack}) {
    const buttonClasses = classNames(styles.button, , {
       [styles.black]: isBlack,
    });

   return <button className={buttonClasses}>{children}</button>
}
```

Reply:
fixed